ui/src/ducks/app/solutions.test.js

```diff
@@ -223,7 +223,9 @@ it('patch namespace with solution when preparing the evironment', () => {
   ];
   expect(gen.next().value.type).toEqual('SELECT');
   const clusterVersion = '2.6.0-dev';
-  expect(gen.next(existingEnv).value).toEqual(
+  expect(gen.next(existingEnv).value.type).toEqual('SELECT');
+  const jobs = [];
+  expect(gen.next(jobs).value).toEqual(
```

Comment:
Updating a test should come in the same commit as the one changing the behaviour of the tested code, IMO.

Reply:
The test changes need to be removed.